src/views/accounts/RiskDetect/AccountCheckTaskList.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <GenericListTable ref="listTable" :header-actions="headerActions" :table-config="tableConfig" />
+  <GenericListTable
+    ref="listTable"
+    :header-actions="headerActions"
+    :table-config="tableConfig"
+    :resource="$t('DetectTasks')"
+  />
 </template>
 
 <script>
```

Comment:
The given code has some syntax errors which I will correct before discussing further. Here's the corrected code:

Before making changes, please review the new code snippet:

```jsx
<template>
     <GenericListTable ref="listTable" :header-actions="headerActions" :table-config="tableConfig" />
</template>


<script>

<script>